LanguageFeatures/Static-access-shorthand/grammar_A05_t01.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion We introduce grammar productions of the form:
+/// ```
+/// <postfixExpression> ::= ...            -- all current productions
+///     | <staticMemberShorthand>          -- added production
+///
+/// <constantPattern> ::=  ...             -- all current productions
+///     | <staticMemberShorthandValue>     -- No selectors, no `.new`.
+///
+/// <staticMemberShorthand> ::= <staticMemberShorthandHead> <selector>*
+///
+/// <staticMemberShorthandHead> ::=
+///       <staticMemberShorthandValue>
+///     | '.' 'new'                                       -- shorthand unnamed constructor
+///
+/// <staticMemberShorthandValue> ::=                      -- something that can potentially create a value.
+///     | '.' <identifier>                                -- shorthand for qualified name
+///     | 'const' '.' (<identifier> | 'new') <arguments>  -- shorthand for constant object creation
+///  ```
+///
+/// @description Checks that it is a compile-time error to use the static access
+/// short syntax with the unary prefix `!` operator.
```

Comment:
Let's just keep it at 'shorthand' again. I think it would be standard terminology to say 'prefix operator' rather than 'unary prefix operator' because 'prefix' implies 'unary'. Did you encounter 'unary prefix operator' somewhere?
```suggestion
/// shorthand syntax with the prefix `!` operator.
```